tests/test_helpers.py

```diff
@@ -474,62 +474,44 @@
       "--date", "2020-04-29", "--collaborators", "afriend", "--description", "more work",
       "--group-members", "ascopatz", "--grants", "SymPy-1.1", "--due-date", "2021-01-01", '--notes', 'new note'],
      "ly_newprojectum has been added in projecta\n"),
-    (["helper", "u_milestone", "sb_firstprojectum", "--index", "4,5",
-      "--status", "converged", "--due-date", "2020-06-01", "--notes", "do this",
-      "do that", "--type", "meeting"],
-     "sb_firstprojectum has been updated in projecta\n"
-     ),
-    (["helper", "u_milestone", "sb"],
+    (["helper", "u_milestone", "--uuid_fragment", "sb_fir", "--status", "finished",
+      "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+     "sb_firstprojectum has been updated in projecta.\n"
+    ),
+    (["helper", "u_milestone", "--uuid_fragment", "kosb_f", "--status", "finished",
```

Comment:
Which lines of code/behavior is this testing that is different from the test above?  Also, fyi, we can usually only have one test that updates the db file, so we simply try and update as many things as possible to test as many lines of code as possible.  It is not great, but ok.

Reply:
The general rule is to make one test that covers the most lines of code.  We have started to make tests for the other functionality but commented out.  Codecov will show which lines of code remain uncovered by the test, but they will be blocks that look like
```
    if rc.something:
        do something
```
and this if is not satisfied in any live test.  Just add as much as you can into one test and we have to hold our fingers for the other things that are not tested.

Reply:
Ok. One more question. 

The `--index` option will no longer be useful when we remove the listing capabilities of `u_milestone`. Do we still want the ability to update multiple milestones at the same time? For example, updating the deliverable and kickoff milestone by entering a list of the milestone uuids? Doing this would also allow us to validate two different parts of the code in one helper test.

Reply:
> Ok. One more question.
> 
> The `--index` option will no longer be useful when we remove the listing capabilities of `u_milestone`. Do we still want the ability to update multiple milestones at the same time? For example, updating the deliverable and kickoff milestone by entering a list of the milestone uuids? Doing this would also allow us to validate two different parts of the code in one helper test.

yes, get rid of --index

for updating multiple ones, I guess why not (especially finishing milestones this is very useful).  So let's have `-u` take a list.

Reply:
Awesome. Will do.